source/index.html

```diff
@@ -3,19 +3,26 @@
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Пинк</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;700&display=swap" rel="stylesheet">
     <link href="css/style.css" rel="stylesheet">
+    <link rel="icon" href="favicon.ico">
+    <link rel="icon" href="source/img/favicons/icon.svg" type="image/svg+xml">
+    <link rel="apple-touch-icon" href="source/img/favicons/180.png">
```

Comment:
```suggestion
    <link rel="apple-touch-icon" href="/img/favicons/apple-touch-icon.png">
```

у всех иконок закреплены названия, потому что они используются браузерами для вставки в свой интерфейс